pkg/eval/mods/file/file.go

```diff
@@ -0,0 +1,100 @@
+// File module which is an equivalent of POSIX like commands.
+// Checkout issule number 1263 for more information.
+
+package file
+
+import (
+	"os"
+	"src.elv.sh/pkg/eval"
+	"src.elv.sh/pkg/eval/vals"
+)
+
+// elvdoc:fn fopen
```

Comment:
In this module the "f" prefix is redundant. It should just be "open"; i.e., "file:open". Same for the function definition and the entry in the namespace table. Similarly for "fclose".